deploy/crds/saas.3scale.net_zyncs_crd.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: zyncs.saas.3scale.net
+spec:
+  group: saas.3scale.net
+  names:
+    kind: Zync
+    listKind: ZyncList
+    plural: zyncs
+    singular: zync
+  scope: Namespaced
+  subresources:
+    status: {}
+  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+  validation:
+    openAPIV3Schema:
+      type: object
+      description: Zync is the Schema for a Zync instance
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+          description: ZyncSpec defines the desired state of Zync
+          required:
+            - secret
+          properties:
+            image:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: Image name (docker repository)
+                version:
+                  type: string
+                  description: Image version
```

Comment:
```suggestion
                  description: Image tag
```

Reply:
Renamed in https://github.com/3scale/saas-operator/pull/5/commits/543f0df2ab75d68d8bf5910559cb2d0c6bec25f0